python/paddle/base/dygraph/base.py

```diff
@@ -449,7 +449,7 @@ def is_grad_enabled() -> bool:
         bool: True if current dygraph gradient calculation mode is enabled, otherwise false.
```

Comment:
```suggestion
   Returns whether current gradient calculation mode is enabled.

   Returns:
       bool: True if current gradient calculation mode is enabled, otherwise false.
```

本 API 已经是动静统一，不再是仅限动态图